content/docs/addons.md

```diff
@@ -4,46 +4,46 @@ title: Add-Ons
 
-- [`PureRenderMixin`](/docs/pure-render-mixin.html). Use [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
-- [`shallowCompare`](/docs/shallow-compare.html), a helper function that performs a shallow comparison for props and state in a component to decide if a component should update. We recommend using [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) instead.
-- [`update`](/docs/update.html). Use [`kolodny/immutability-helper`](https://github.com/kolodny/immutability-helper) instead.
-- [`ReactDOMFactories`](https://www.npmjs.com/package/react-dom-factories), pre-configured DOM factories to make React easier to use without JSX.
+- [`PureRenderMixin`](/docs/pure-render-mixin.html). Bunun yerine [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) kullanın.
+- [`shallowCompare`](/docs/shallow-compare.html), bir componentin güncellenip güncellenmeyeceğine karar vermek için componentin props ve state'lerini yüzeysel olarak karşılaştıran bir yardımcı işlev. Bunun yerine  [`React.PureComponent`](/docs/react-api.html#reactpurecomponent) kullanılmasını tavsiye ederiz.
+- [`update`](/docs/update.html). Bunun yerine [`kolodny/immutability-helper`](https://github.com/kolodny/immutability-helper) kullanın.
+- [`ReactDOMFactories`](https://www.npmjs.com/package/react-dom-factories), React'in JSX olmadan kullanımını kolaylaştırmak için önceden yapılandırılmış DOM factory'leri.
 
-### Deprecated Add-ons {#deprecated-add-ons}
+### Kullanımdan Kaldırılan Eklentiler {#deprecated-add-ons}
 
-- [`LinkedStateMixin`](/docs/two-way-binding-helpers.html) has been deprecated.
-- [`TransitionGroup` and `CSSTransitionGroup`](/docs/animation.html) have been deprecated in favor of [their drop-in replacements](https://github.com/reactjs/react-transition-group/tree/v1-stable).
+- [`LinkedStateMixin`](/docs/two-way-binding-helpers.html) kullanımdan kaldırılmıştır.
+- [`TransitionGroup` and `CSSTransitionGroup`](/docs/animation.html)  [yerlerine geçen eklentilerin](https://github.com/reactjs/react-transition-group/tree/v1-stable) kullanılması için kullanımdan kaldırılmışlardır.
 
-## Using React with Add-ons {#using-react-with-add-ons}
+## React'i Eklentilerle Kullanma {#using-react-with-add-ons}
 
-You can install the add-ons individually from npm (e.g. `npm install react-addons-create-fragment`) and import them:
+Eklentileri NPM aracılığyla ayrı ayrı yükleyebilir (örnek: `npm install react-addons-create-fragment`) ve onları içeri aktarabilirsiniz:
 
 ```javascript
 import createFragment from 'react-addons-create-fragment'; // ES6
 var createFragment = require('react-addons-create-fragment'); // ES5 with npm
 ```
 
-When using React 15 or earlier from a CDN, you can use `react-with-addons.js` instead of `react.js`:
+When using React 15 or earlier from a CDN, you can use React 15 ya da daha önceki sürümleri bir CDN'den kullanırken `react.js` yerine  `react-with-addons.js` kullanabilirsiniz:
```

Comment:
I think you forgot to delete the English part here.

Reply:
Yes, @alioguzhan, good call! I skipped the delete English part. I'll fix it now.